pkg/core/consensus/user/sortition.go

```diff
@@ -5,12 +5,16 @@ import (
 	"math"
 	"math/big"
 
-	"github.com/dusk-network/dusk-blockchain/pkg/core/data/wallet"
 	"github.com/dusk-network/dusk-blockchain/pkg/util/nativeutils/sortedset"
 	"github.com/dusk-network/dusk-crypto/hash"
 	log "github.com/sirupsen/logrus"
 )
 
+// DUSK is one whole unit of DUSK. This is duplicated from wallet since
+// otherwise we get into an import cycle including the transactions and users
+// packages
+const DUSK = uint64(100000000)
```

Comment:
Could we avoid cyclic imports and duplication by just putting this in the `config` package?

Reply:
yes, this was a quick hack. Did not give it that much thought

Reply:
will open a separate ticket for it